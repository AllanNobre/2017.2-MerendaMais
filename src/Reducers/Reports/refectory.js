const refectory = [
  { label: 'Escola possui Refeitório', markedYes: false, markedNo: false, status: false, key: 0 },
  { label: 'Alimentos prontos sem proteção', markedYes: false, markedNo: false, status: false, key: 1 },
  { label: 'Refeição sendo servida sem talheres ', markedYes: false, markedNo: false, status: false, key: 2 },
  { label: 'Alunos compartilhando os copos no bebedouro', markedYes: false, markedNo: false, status: false, key: 3 },
  { label: 'Pia sem sabonete e papel toalha para lavar as mãos', markedYes: false, markedNo: false, status: false, key: 4 },
  { label: 'Não vi pia', markedYes: false, markedNo: false, status: false, key: 5 },
  { label: 'Telhado sem forro', markedYes: false, markedNo: false, status: false, key: 6 },
  { label: 'Telhado com furos', markedYes: false, markedNo: false, status: false, key: 7 },
  { label: 'Lixeiras destampadas', markedYes: false, markedNo: false, status: false, key: 8 },
  { label: 'Chão difícil de limpar', markedYes: false, markedNo: false, status: false, key: 9 },
  { label: 'Parede difícil de limpar', markedYes: false, markedNo: false, status: false, key: 10 },
  { label: 'Janelas sem tela de proteção', markedYes: false, markedNo: false, status: false, key: 11 },
  { label: 'Portas sem tela de proteção', markedYes: false, markedNo: false, status: false, key: 12 },
  { label: 'Portas sem rodapé', markedYes: false, markedNo: false, status: false, key: 13 },
  { label: 'Insetos sobrevoando o local', markedYes: false, markedNo: false, status: false, key: 14 },
  { label: 'Ventilador de teto empoeirado', markedYes: false, markedNo: false, status: false, key: 15 },
  { label: 'Local mal iluminado', markedYes: false, markedNo: false, status: false, key: 16 },
  { label: 'Local sujo', markedYes: false, markedNo: false, status: false, key: 17 },
];

export default refectory;
