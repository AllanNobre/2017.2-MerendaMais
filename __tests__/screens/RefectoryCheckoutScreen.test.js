import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import configureStore from 'redux-mock-store';
// imported as a connected component!
import RefectoryCheckoutContainer from '../../src/Containers/RefectoryCheckoutContainer';
import refectory from '../../src/Reducers/Reports/refectory';

Enzyme.configure({ adapter: new Adapter() });

const mockStore = configureStore();

const initialState = {
  report: {
    refectory,
    foodQualityObservation: '',
  },
};

const store = mockStore(initialState);

describe('Testing ReportObservation Screen', () => {
  it('renders as expected', () => {
    const wrapper = shallow(
      <RefectoryCheckoutContainer />,
      { context: { store } },
    ).dive();
    expect(wrapper).toMatchSnapshot();
  });
});
