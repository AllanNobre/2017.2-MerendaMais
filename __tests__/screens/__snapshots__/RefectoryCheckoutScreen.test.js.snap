// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing RefectoryCheckout Screen renders as expected 1`] = `
<ScrollViewMock
  style={
    Object {
      "backgroundColor": "white",
      "flex": 6,
      "flexDirection": "column",
      "marginBottom": 9,
    }
  }
>
  <Header
    backButton={true}
    subTitle="Refeitório"
    title="Relatório"
  />
  <View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "fontWeight": "bold",
            "paddingLeft": 10,
            "paddingTop": 5,
          }
        }
      >
        Sim
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "fontWeight": "bold",
            "paddingLeft": 20,
            "paddingTop": 5,
          }
        }
      >
        Nao
      </Text>
    </View>
    <View
      key="0"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Escola possui Refeitório
      </Text>
    </View>
    <View
      key="1"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Alimentos prontos sem proteção
      </Text>
    </View>
    <View
      key="2"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Refeição sendo servida sem talheres 
      </Text>
    </View>
    <View
      key="3"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Alunos compartilhando os copos no bebedouro
      </Text>
    </View>
    <View
      key="4"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Pia sem sabonete e papel toalha para lavar as mãos
      </Text>
    </View>
    <View
      key="5"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Não vi pia
      </Text>
    </View>
    <View
      key="6"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Telhado sem forro
      </Text>
    </View>
    <View
      key="7"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Telhado com furos
      </Text>
    </View>
    <View
      key="8"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Lixeiras destampadas
      </Text>
    </View>
    <View
      key="9"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Chão difícil de limpar
      </Text>
    </View>
    <View
      key="10"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Parede difícil de limpar
      </Text>
    </View>
    <View
      key="11"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Janelas sem tela de proteção
      </Text>
    </View>
    <View
      key="12"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Portas sem tela de proteção
      </Text>
    </View>
    <View
      key="13"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Portas sem rodapé
      </Text>
    </View>
    <View
      key="14"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Insetos sobrevoando o local
      </Text>
    </View>
    <View
      key="15"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Ventilador de teto empoeirado
      </Text>
    </View>
    <View
      key="16"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Local mal iluminado
      </Text>
    </View>
    <View
      key="17"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#008000"
        />
      </View>
      <View>
        <Checkbox
          checkboxStyle={
            Object {
              "borderColor": "black",
              "borderRadius": 2,
              "borderWidth": 2,
              "flexWrap": "wrap",
              "height": 25,
              "marginLeft": 10,
              "marginRight": 10,
              "marginTop": 10,
              "width": 25,
            }
          }
          defaultColor="#fff"
          disabled={false}
          disabledColor={null}
          onSelect={[Function]}
          selected={false}
          selectedColor="#B22222"
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "fontSize": 15,
            "paddingTop": 15,
          }
        }
      >
        Local sujo
      </Text>
    </View>
  </View>
  <View
    behavior="padding"
  >
    <View
      style={
        Object {
          "paddingLeft": 10,
          "paddingRight": 10,
          "paddingTop": 30,
        }
      }
    >
      <TextInput
        multiline={true}
        onChangeText={[Function]}
        placeholder="Observações (opcional)"
        style={
          Object {
            "borderRadius": 12,
            "borderWidth": 1,
            "fontSize": 37.5,
            "height": 333.5,
            "paddingLeft": 10,
            "textAlignVertical": "top",
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
  </View>
  <TouchableOpacity
    activeOpacity={0.2}
    key="setObservation"
    onPress={[Function]}
    style={
      Object {
        "backgroundColor": "#FF9500",
        "borderRadius": 7,
        "borderWidth": 1,
        "justifyContent": "flex-end",
        "marginBottom": 20,
        "marginHorizontal": 15,
        "marginTop": 30,
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFF",
          "textAlign": "center",
        }
      }
    >
      Concluir
    </Text>
  </TouchableOpacity>
</ScrollViewMock>
`;
